engine/installation/linux/docker-ee/ubuntu.md

```diff
@@ -0,0 +1,270 @@
+
+- Most users
+  [set up Docker's repositories](#install-using-the-repository) and install
+  from them, for ease of installation and upgrade tasks. This is the
+  recommended approach.
+
+- Some users download the DEB package and install it manually and manage
+  upgrades completely manually. This is useful in situations such as installing
+  Docker on air-gapped systems with no access to the internet.
+
+### Install using the repository
+
+Before you install Docker EE for the first time on a new host machine, you need
+to set up the Docker repository. Afterward, you can install and update Docker EE
+from the repository.
+
+#### Set up the repository
+
+1.  Install packages to allow `apt` to use a repository over HTTPS:
+
+  ```bash
+  $ sudo apt-get install \
+      apt-transport-https \
+      ca-certificates \
+      curl \
+      software-properties-common
+  ```
+
+2.  Add Docker's official GPG key using your customer Docker EE repository URL:
+
+  ```bash
+  $ curl -fsSL <DOCKER-EE-URL>/gpg | sudo apt-key add -
+  ```
+
+  Verify that the key fingerprint is `DD91 1E99 5A64 A202 E859  07D6 BC14 F10B 6D08 5F96`.
+
+  ```bash
+  $ apt-key fingerprint 0EBFCD88
+
+  pub   4096R/6D085F96 2017-02-22
+      Key fingerprint = DD91 1E99 5A64 A202 E859  07D6 BC14 F10B 6D08 5F96
+  uid       [ultimate] Docker Release (EE deb) <docker@docker.com>
+  sub   4096R/91A29FA3 2017-02-22
+  ```
+
+3.  Use the following command to set up the **stable** repository, replacing
+  `<DOCKER-EE-URL>` with the URL you noted down in the
+  [prerequisites](#prerequisites).
+
+  > **Note**: The `lsb_release -cs` sub-command below returns the name of your
+  > Ubuntu distribution, such as `xenial`.
+  >
+
+  ```bash
+  $ sudo add-apt-repository \
+     "deb [arch=amd64] <DOCKER-EE-URL> \
+     $(lsb_release -cs) \
+     stable-{{ minor-version }}"
+  ```
+
+#### Install Docker EE
+
+1.  Update the `apt` package index.
+
+    ```bash
+    $ sudo apt-get update
+    ```
+
+2.  Install the latest version of Docker EE, or go to the next step to install a
+    specific version. Any existing installation of Docker EE is replaced.
+
+    Use this command to install the latest version of Docker EE:
+
+    ```bash
+    $ sudo apt-get install docker-ee
+    ```
+
+
+    > **Warning**:
+    > If you have multiple Docker repositories enabled, installing
+    > or updating without specifying a version in the `apt-get install` or
+    > `apt-get update` command will always install the highest possible version,
+    > which may not be appropriate for your stability needs.
+    {:.warning}
+
+3.  On production systems, you should install a specific version of Docker EE
+    instead of always using the latest. This output is truncated. List the
+    available versions.
+
+    ```bash
+    $ apt-cache madison docker-ee
+
+    docker-ee | {{ minor-version }}.0~ee-0~ubuntu-xenial | {{ download-url-base}} xenial/stable amd64 Packages
```

Comment:
Same here for pinning to minor releases, although I just realize that I don't know if Docker EE has a similar directory structure as CE in the repository